homeassistant/components/pi_hole/config_flow.py

```diff
@@ -101,9 +116,11 @@ async def async_step_init(self, user_input, is_import=False):
                         CONF_LOCATION,
                         default=user_input.get(CONF_LOCATION) or DEFAULT_LOCATION,
                     ): str,
-                    vol.Optional(
-                        CONF_API_KEY, default=user_input.get(CONF_API_KEY) or ""
-                    ): str,
+                    vol.Required(
+                        CONF_STATISTICS_ONLY,
+                        default=user_input.get(CONF_STATISTICS_ONLY)
```

Comment:
default=user_input.get(CONF_STATISTICS_ONLY, DEFAULT_STATISTICS_ONLY )
```suggestion
                        default=user_input.get(CONF_STATISTICS_ONLY, DEFAULT_STATISTICS_ONLY )
```